int-tests/007-update.t

```diff
@@ -0,0 +1,29 @@
+This cram test performs the update command on one slice.
```

Comment:
Anyway it clarifies what the setup looks like. There is enormous potential to get the concepts wrong and create misunderstanding IMO.